### ISSUE TITLE
Nuxt import component: Prefix doesn't work

### ISSUE THREAD

**Issue by FabienVINCENT** [primefaces/primevue#6051]
### Describe the bug

When you configure the prefix with the nuxt module, the components are not imported

### Reproducer

https://stackblitz.com/edit/github-tvge24?file=nuxt.config.ts

### PrimeVue version

4.0.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Nuxt

### Browser(s)

Arc

### Steps to reproduce the behavior

Remove the prefix configuration and the button with label "No prefix" work

**Comment by Tarabass** [comment]
Try `Prime` instead of `prime`:

https://stackblitz.com/edit/github-tvge24-4nqjsd?file=nuxt.config.ts

**Comment by FabienVINCENT** [comment]
Why i'm need to manually import with `<component is` tag's ?

when I uncomment both are working, but when commented only the primary is working
`<template>
    <component is="PrimeButton" label="Component" />
    <PrimeButton label="Primary" />
</template>

<script setup>
//import Button from 'primevue/button';
</script>
`

**Comment by Tarabass** [comment]
There are [two ways](https://nuxt.com/docs/guide/directory-structure/components#dynamic-components) to use dynamic components. I prefer the second one. In short, you have to resolve the component first because the component is not resolving it for you.

```js
<script setup lang="ts">
import { SomeComponent } from '#components'

const MyButton = resolveComponent('MyButton')
</script>

<template>
  <component :is="clickable ? MyButton : 'div'" />
  <component :is="SomeComponent" />
</template>
```

But did my first answer helped you? Before we go through all functionality of Vue or Nuxt, while we are here for PrimeVue.

**Comment by sparrow-chik-chrk** [comment]
Indeed, auto-import stopped working properly when upgrading from version v4.0.0-rc3 to v4.0.0. Not only did the prefixes break, but also the styles.

**Comment by FabienVINCENT** [comment]
@Tarabass Yes it works when I do an explicit import but the auto import should work. In any case in 4.0.0-rc.3 it worked without explicit import

**Comment by FabienVINCENT** [comment]
> Indeed, auto-import stopped working properly when upgrading from version v4.0.0-rc3 to v4.0.0. Not only did the prefixes break, but also the styles.

Yes indeed in 4.0.0-rc.3 it worked without problem

**Comment by Mychen3** [comment]
4.0.1  I still have this problem

**Comment by sparrow-chik-chrk** [comment]
@mertsincan 
It doesn't work not only with <component is=. In general, auto-import doesn't add components until you explicitly import them. On the rc version, it works stably.

**Comment by Tarabass** [comment]
> @mertsincan It doesn't work not only with <component is=. In general, auto-import doesn't add components until you explicitly import them. On the rc version, it works stably.

Why is it working in my stackblitz, primevue 4, or am I missing something?

https://stackblitz.com/edit/github-tvge24-4nqjsd?file=nuxt.config.ts

**Comment by mertsincan** [comment]
@Tarabass +1 ;) The logic of auto-import is to be able to use components without manually importing them. In RC and other older versions, nuxt-module added all components(80+) directly and users had to set up treeshaking manually. With the autoImport feature, we automatically register components using the unplugin API.

**Comment by reitowo** [comment]
So what's the expected behavior? The latest version still not work with `prefix` + `autoImport`. If they are meant to not working together, perhaps leave some words in doc?

**Comment by Erdu-EC** [comment]
I have the same problem, autoimport not working with prefix set to "Prime".

![image](https://github.com/user-attachments/assets/81e9e3f0-ad8f-451e-8ab0-64b23ec9b5ff)

When i remove the "Prime" from component it work, but not is the best behaviour for the project.

![image](https://github.com/user-attachments/assets/b1b937e4-abab-485d-9e0b-5ab1c34e48b8)

**Comment by vincenzomartusciello** [comment]
I have the same behaviour @Erdu-EC 
From v4 the entries in .nuxt/components.d.ts aren't generated, but on production it works.

**Comment by sparrow-chik-chrk** [comment]
@vincenzomartusciello It's still a problem, even if it works in production. It creates significant inconveniences during development.

**Comment by jmroon** [comment]
Trying PrimeVue for the first time. Running into the same issue. Bit of a dealbreaker, unfortunately.

**Comment by tugcekucukoglu** [comment]
Please try to use the latest version. If the problem persists, please reopen the issue.
